recipes/octo-logger-cpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::octo-logger-cpp)
```

```suggestion
target_link_libraries(${PROJECT_NAME} octo-logger-cpp::octo-logger-cpp)
```
CONAN_PKG will be deprecated soon.

---

didnt know about that, fixed :)